src/utils/case_utils/case_columns.js

```diff
@@ -65,6 +65,7 @@ export function case_columns(
       ),
     },
     {
+
```

Comment:
I see you added a blank line here, I might add a blank line at the end of this code block for consistency/readability's sake, or I would get rid of this line altogether.